packages/api-client/src/connections/CentralApi.ts

```diff
@@ -36,4 +36,12 @@ export class CentralApi extends BaseApi {
       await this.connection.post('surveyResponse', null, chunk);
     }
   }
+
+  /**
+   * Solely for use by meditrak-app-server. Should be deprecated once RN-556 is implemented
+   * @param changes Changes from the meditrak-app sync
+   */
+  public async meditrak_only_pushChanges(changes: unknown[]): Promise<{ message: string }> {
```

Comment:
Taking a leaf out of @IgorNadj 's book, I'm very explicitly naming this function to ensure it doesn't get used inappropriately 😉